src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -477,15 +571,47 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
       }
 
       people = this._groupPeople || [];
+      if (people) {
+        people = people.filter((person: IDynamicPerson) => {
+          return person.displayName.toLowerCase().indexOf(input) !== -1;
```

Comment:
I realize this is moved code from before this change, but it's possible that the input is intended to search more than just the display name

Reply:
Yeah, this filter is only when `group-id` is set, so it's filtering against the group members. Updated.